.github/workflows/ui-tests/cypress/integration/login_spec.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe('Login Page', function() {
+  it('successfully loads', function() {
+    cy.visit('/login')
+  })
+})
+
+describe('Default Credentials on Form Submission', function() {
+  const username = 'testuser'
```

Comment:
@murog these should be pre filled based on `value="testuser"` right? Would it not be better to retrieve that value and use it so we know that the pre filling also works as intended.

Reply:
good catch.  I think it's best to keep the two things decoupled, so a test for verifying the pre-filled username and password are what we expect and tests for login of our default credentials